GliaWidgets/Sources/Extensions/UIColor+Hex.swift

```diff
@@ -27,7 +27,7 @@ extension UIColor {
         self.init(red: components.R, green: components.G, blue: components.B, alpha: components.a)
     }
 
-    func toHex(alpha: Bool = false) -> String? {
+    func toHex(alpha: Bool = true) -> String? {
```

Comment:
Why is this needed?

Reply:
We have one color with `0.5` alpha component value in default color pallet. So we need to be able transform it to hex without losing `alpha`. 
We have `UIColor.hex` computed property to convert UIColor instance to hex-string value. But it omits `alpha` component if it exists. I found this `toHex` method that was added and has never been used in Widgets. So I changed default value to `true`, because it does not affect other places.

Reply:
`toHex` is not very self explanatory name, because it not possible to get what kind of color hex representation it is (is it RGB/RGBA, ARGB?). If it is RGB hex value, then it has to be something like `toRGBHex`. Does it make sense?

Reply:
Fixed 28138aa40ffb224181edf1b751a612cf2c061f01